projects/centre_console/test/test_brake_light_control.c

```diff
@@ -0,0 +1,86 @@
+#include "brake_light_control.h"
+
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "centre_console_events.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "log.h"
+#include "ms_test_helper_can.h"
+#include "ms_test_helpers.h"
+#include "pedal_rx.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+typedef enum {
+  TEST_CAN_BRAKE_LIGHT_EVENT_TX = 0,
+  TEST_CAN_BRAKE_LIGHT_EVENT_RX,
+  TEST_CAN_BRAKE_LIGHT_EVENT_FAULT,
+  NUM_TEST_CAN_EVENTS,
+} TestCanEvent;
+
+static CanStorage s_can_storage = { 0 };
+
+// Stores unpacked data from RX handler messages
+static uint8_t s_brake_lights_id;
+static uint8_t s_brake_lights_state;
```

Comment:
You should reset these to some invalid value inside `setup_test` so that their values at the end of one test don't affect subsequent tests.